### PR TITLE
[#132858443] No subtotal in Referral webhook payload

### PR DESCRIPTION
**Pull request by dgtv**

### CODE REVIEW

File: source/web_hooks/referral.rst [hunk 1]
```diff
@@ -153,7 +155,7 @@ Triggered when there is a new referral in Talkable.
...
...9610,"type":"StandaloneCampaign","cached_slug":"affiliate-campaign-test","tag_names":["default"]},"offer":{"email":"referrer@example.com","short_url_code":"1a2PV"},"referred_origin":{"id":543208538,"type":"Purchase","order_number":381177444,"subtotal": 13.43,"customer_id":"22726076","coupon_code":"WHT60000"},"advocate_rewards":[{"id":316145153,"email":"referrer@example.com","person":{"email":"referrer@example.com","first_name":"Bob","last_name":"Crane","gender":null,"sub_choice":false,"subscribed_at":null,"unsubscribed_at":null},"amount":"5.00","incentive":"rebate","incentive_description":"$5.00 back","origin":{"id":667277390,"type":"AffiliateMember","email":"referrer@example.com"}}],"friend_rewards":[{"id":884178819,"email":"referred@example.com","person":{"email":"referred@example.com","first_name":"Alice","last_name":"Smith","gender":"female","sub_choice":true,"subscribed_at":"2014-08-14T02:01:16.823-07:00","unsubscribed_at":null},"amount":"0.00","incentive":"other","incentive_description":"First Month Free","origin":{"id":543208538,"type":"Purchase","order_number":381177444,"order_date":"2014-08-14T02:01:16.824-07:00","customer_id":"22726076","coupon_code":"WHT60000"}}]}' <url>
```
Comment (Tonkonozhenko) [inline source/web_hooks/referral.rst]: Please make it more readable

Comment (dgtv) [inline source/web_hooks/referral.rst]: @Tonkonozhenko I made it in the last commit but if your terminal window has less than `n` lines (where `n` is num of lines of curl command) you won't be able to just copy and paste this command to your terminal. So we should choose between readable and convenient.

File: source/web_hooks/referral.rst [hunk 2]
```diff
@@ -153,7 +155,29 @@ Triggered when there is a new referral in Talkable.
...
+   curl <url> \
+     -d 'key=<key>&payload={
+     "campaign": {
```
Comment (Tonkonozhenko) [inline source/web_hooks/referral.rst]: Does it work? I suppose that we need slashes at the end of each line

Comment (dgtv) [inline source/web_hooks/referral.rst]: yep, curl automatically removes `\n` symbols

**Pull request merged**